frontend/components/Status.tsx

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ITask, Ilist } from "../src/App";
+
+function Status(props: {
+  submited: boolean;
+  setSubmited: (Submit: boolean) => void;
+  list: ITask[];
+  setList: (list: ITask[]) => void;
+}) {
+  const [complete, setComplete] = useState<ITask[]>([]);
+  const [pending, setPending] = useState<ITask[]>([]);
+  function splitList() {
```

Comment:
I'd rename this one - the name `splitList` describes the programming operation, or in other words, how you achieve the goal. Naming should reflect _what_ is your goal instead of _how_ you implemented it. In this case this could be something like `divideTasksAccordingToTheirCompletedState`. Feel free to come up with your own if mine seems too wordy.

Reply:
I could be wrong as I have not been able to get it running locally, but I feel like we can remove this function altogether. 

From what I can tell, we do not need to store `completedTasks` and `pendingTasks` in state, we just want to take the `taskList` and filter depending on the value of `completed`. I would do this directly in with the JSX in the return statement.

See my code snippet the comment further down in this file.